src_base/xeve_bsw.c

```diff
@@ -198,7 +198,7 @@ int xeve_bsw_write(XEVE_BSW * bs, u32 val, int len) /* len(1 ~ 32) */
     int leftbits;
 
     xeve_assert(bs);
-    xeve_assert(len <= 32); // to avoid shifting by a negative value
+    //xeve_assert(len <= 32); // to avoid shifting by a negative value
```

Comment:
Describe why this change is required?

Reply:
If adaptive quantization method functions are enabled then I receive error.

Reply:
Proper way is to solve issue not disable assert.

Reply:
Closing change as invalid.
If you have problems with adaptive quantization please open issue with description and data that allow to find root cause of problem.